hs-bindgen/src-internal/HsBindgen/Frontend.hs

```diff
@@ -110,15 +116,16 @@ processTranslationUnit
 
     let (afterSort, msgsSort) =
           sortDecls afterParse
+        sliceConfig = SliceConfig configProgramSlicing configPredicate
```

Comment:
Let's not define this here, it breaks the flow, and it suggests that the config may depend on the results of the previous passes. Let's move it to the `where` clause.